spring-boot-test-autoconfigure/src/test/resources/application.properties

```diff
@@ -0,0 +1,2 @@
+# Use random port for embedded Mongod.
+spring.data.mongodb.port = 0
```

Comment:
Isn't that a smell? I mean, if that's related to `DataMongoTest` in any way, it should do that automatically, isn't it?